### DISCUSSION TITLE
Weekly backup

### DISCUSSION THREAD

**Discussion by alexios-y** [standardnotes/forum#2766] [Suggestions]
Right now I find it's too frequent to have a daily backup. It would be ideal to have a weekly backup. Is it possible to have an option so users can choose whatever frequency they want?

**Comment by redpay** [comment]
This is something I would use too, it doesn´t bother me too much receiving daily backups by email but I only need it weekly, I don´t work on my notes daily, I think it would save SN server resources having to email less often.

**Comment by myreli** [comment]
You can now choose between daily and weekly backups under Preferences > Backups > Email Backups

<img width="890" alt="image" src="https://user-images.githubusercontent.com/17554234/220431058-94df1027-a486-4302-960e-ddac43dd88e2.png">
